specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -0,0 +1,1534 @@
+            "$ref": "#/definitions/Database"
+          },
+          "description": "The list of Kusto databases."
+        }
+      },
+      "description": "The list Kusto databases operation response."
+    },
+    "Database": {
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location."
+        },
+        "kind": {
+          "description": "Kind of the database",
+          "enum": [
+            "ReadWrite",
+            "ReadOnlyFollowing"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Class representing a Kusto database.",
+      "discriminator": "kind"
+    },
+    "ReadWriteDatabase": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The database properties.",
+          "$ref": "#/definitions/ReadWriteDatabaseProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Database"
+        }
+      ],
+      "description": "Class representing a read write database.",
+      "x-ms-discriminator-value": "ReadWrite"
+    },
+    "ReadWriteDatabaseProperties": {
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ResourceProvisioningState",
+          "description": "The provisioned state of the resource."
+        },
+        "softDeletePeriod": {
+          "type": "string",
+          "format": "duration",
+          "description": "The time the data should be kept before it stops being accessible to queries in TimeSpan."
+        },
+        "hotCachePeriod": {
+          "type": "string",
+          "format": "duration",
+          "description": "The time the data should be kept in cache for fast queries in TimeSpan."
+        },
+        "statistics": {
+          "$ref": "#/definitions/DatabaseStatistics",
+          "description": "The statistics of the database."
+        },
+        "isFollowed": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Indicates whether the database is followed."
+        }
+      },
+      "description": "Class representing the Kusto database properties."
+    },
+    "DatabaseStatistics": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "size": {
+          "type": "number",
```

Comment:
>"type": "number", [](start = 10, length = 17)

consider adding "format" for the number type

Reply:
Advised with other members of our team, but we preferred to imitate the _kusto.json_ resource and currently skip adding the format for number type in order to avoid unexpected behavior of it.